libr/anal/p/pyc.mk

```diff
@@ -0,0 +1,40 @@
+OBJ_PYC=anal_pyc.o
+OBJ_PYC+=../../asm/arch/pyc/opcode_10.o
```

Comment:
can you use $(subst or $(addprefix to just construct the filename array here instead of adding the same prefix by hand to all the entries?